src/redux/migrations.ts

```diff
@@ -575,9 +575,7 @@ export const migrations = {
     },
     app: {
       ...state.app,
-      finclusiveUnsupportedStates: REMOTE_CONFIG_VALUES_DEFAULTS.finclusiveUnsupportedStates.split(
-        ','
-      ),
+      finclusiveUnsupportedStates: ['NY', 'TX'],
```

Comment:
This is the same as before, I've just removed `finclusiveUnsupportedStates` from the remote config type so I'm hardcoding it here.